### ISSUE TITLE
Framework info: Channel axis is set to be the same for all the layers in the framework.

### ISSUE THREAD

**Issue by lapid92** [sony/model_optimization#250]
In default_framework_info the channel axis is set to be the same for all the layers in the framework.
While it works for keras (the channel axis of all the layers is last one) for Pytorch it's not true (Conv channel axis is 1 and for Linear is the last one).
Maybe this needs to be a Dict with value per layer.
